limitador-server/src/main.rs

```diff
@@ -785,10 +801,26 @@ fn storage_config_from_env() -> Result<StorageConfiguration, ()> {
                 consistency: env::var("INFINISPAN_COUNTERS_CONSISTENCY").ok(),
             },
         )),
-        _ => Ok(StorageConfiguration::InMemory),
+        _ => Ok(StorageConfiguration::InMemory(
+            InMemoryStorageConfiguration { cache_size: None },
+        )),
     }
 }
 
+fn guess_cache_size() -> Option<u64> {
+    let sys = System::new_with_specifics(RefreshKind::new().with_memory());
+    let free_mem = sys.available_memory();
+    let memory = free_mem as f64 * 0.7;
```

Comment:
isn't the 70% too much (or too little xD)

Reply:
Well... this math is "wrong", the size of the entries don't account for the overhead of the memory required by the cache itself... but most importantly don't account for the size of the `variables` data that's on the heap... i.e. all the entries of `name/value` pairs... we need some overhead for that.
Finally the "rest" also needs memory, i.e. the actix & tonic & all other plumbing (if only storing the `Limit`s themselves... So for tiny amounts of `sys.available_memory()` 70% might actually be way too eager, while for large ones, probably on the overly cautious end... Which is why this _will trigger a `WARN` level log_.